listenbrainz/db/mbid_manual_mapping.py

```diff
@@ -87,3 +89,24 @@ def get_mbid_manual_mappings(recording_msid: uuid.UUID) -> List[MbidManualMappin
             }
         )
         return [MbidManualMapping(**row) for row in result.mappings()]
+
+def check_manual_mapping_exists(user_id: int, recording_msids: Iterable[str]) -> set[str]:
+    """Check if a user has a mapping for a list of recordings
+
+    Arguments:
+        user_id: LB user id of a user
+        recording_msids: the msids of the recordings to check
+
+    Returns:
+        A set of msids for which the user has a mapping
+    """
+    query = SQL("""
```

Comment:
Do you bother to open your eyes when writing mundane queries like this or does it just flow from your soul? (or all the souls you've stolen?) :laughing: